models/article_models.js

```diff
@@ -14,4 +14,14 @@ selectArticleById = (article_id) => {
     });
 };
 
-module.exports = { selectArticleById };
+selectAllArticles = () => {
+  return db
+    .query(
+      "SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, CAST(COUNT(comment_id) AS INT) AS comment_count FROM articles LEFT JOIN comments ON comments.article_id = articles.article_id GROUP BY articles.article_id ORDER BY created_at DESC;"
+    )
```

Comment:
Really lovely command! If you use backticks, you'd be able to separate this out onto multiple lines, to make it easier for you to sort through :)